tests/test_outputs.rs

```diff
@@ -0,0 +1,81 @@
+#![cfg(feature = "integration-tests")]
+use std::ffi::OsStr;
+use std::path::Path;
+
+use std::{env, process};
+use tempfile::tempdir;
+
+const NO_ARGS: &[&str] = &[];
+
+fn exe_runtime(
+    package: &str,
+    work_dir: &Path,
+    command: &str,
+    args: impl IntoIterator<Item = impl AsRef<OsStr>>,
+) -> anyhow::Result<process::Output> {
+    let app = env!("CARGO_BIN_EXE_ya-runtime-wasi");
+    let output = process::Command::new(app)
+        .stderr(process::Stdio::piped())
+        .stdout(process::Stdio::piped())
+        .args(&["--task-package", package])
+        .arg("--workdir")
+        .arg(work_dir)
+        .arg(command)
+        .args(args)
+        .output()?;
+    Ok(output)
+}
+
+#[test]
+fn test_outputs() -> anyhow::Result<()> {
+    let package = "tests/trusted-voting-mgr-66d7ce8208f4da9d7cbd.ywasi";
```

Comment:
I'm not a fan of storing binaries in a repo on Github, but if there's no way on compiling the example on the fly, I guess it will have to do.

Reply:
i am not fun either. i was thinking about http resource. i believe that `git-lfs` was good enough choice  in this case.